docs/rules/no-mixed-linebreaks.md

```diff
@@ -0,0 +1,41 @@
+# Disallow mixing CRLF and LF linebreaks (no-mixed-linebreaks)
+
+When developing with a lot of people all having different editors, VCS applications and operating systems it may occur that
+different line endings are written by either of the mentioned (might especially happen when using the windows and mac versions of SourceTree together).
+
+Many versioning systems (like git and subversion) can automatically ensure the correct ending. However to cover all contingencies you can activate this rule.
+
+## Rule Details
+
+This rule aims to ensure having consistent line endings independent of operating system, VCS or editor used.
+
+The following patterns are considered warnings:
+
+```js
+    var a = 'a',\r\n
+        b = 'b';\n
+
+    // option 'LF'
+    var a = 'a';\r\n
+
+    // option 'CRLF'
+    var a = 'a';\n
+```
+
+The following patterns are not warnings:
+
+```js
+    // option 'LF'
+    var a = 'a';\n
+
+    // option 'CRLF'
+    var a = 'a';\r\n
+```
+
+### Options
+
+This rule may take one option which is either `LF` or `CRLF`. When omitted `LF` is assumed.
+
```

Comment:
Yeah, I'm not fond of the JSCS way of doing it. I'd much prefer `\r\n` because it exactly matches people's understanding of those characters.

Another option would be `"windows"` and `"unix"`.